blocks/library/chat/style.scss

```diff
@@ -0,0 +1,20 @@
+.editor-visual-editor__block[data-type="core/chat"] {
```

Comment:
I think we should always use `wp-block- ` unless we need to change something in the parent wrapper (which is unlikely to happen, it's needed for float alignments essentially)